chalicelib/checks/deployment_checks.py

```diff
@@ -193,6 +192,8 @@ def _deploy_application_to_beanstalk(connection, **kwargs):
     if application_version_name is None:  # if not specified, use branch+timestamp
         application_version_name = 'foursight-package-%s-%s' % (branch, datetime.datetime.utcnow())
 
+    # TODO: This is defaulting its arguments. Should it be taking repo dataflow from above where 'repo' var is unused?
+    #       -kmp 5-Oct-2021
```

Comment:
At one point the answer was likely yes but since we split this code it's not needed anymore and I'm guessing I just forgot to get rid of it.